.github/workflows/generate-sbom.yaml

```diff
@@ -38,6 +38,8 @@ jobs:
     steps:
```

Comment:
If you may fix here as well 

```
      - name: Retrieve product.txt from artifacts
        run: >
          curl --fail -LO -u ${{ secrets.ARTIFACTS_USER }}:${{ secrets.ARTIFACTS_PASSWORD }}
          ${{ inputs.artifacts-url }}/product.txt
```

It's what I was thinking of this morning

Reply:
Oh yes, this one